.travis.yml

```diff
@@ -0,0 +1,16 @@
+notifications:
+  email: false
+
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+install:
+  - docker build -t worldpopulation .
+  - docker run -v `pwd`:/code -i -t --rm worldpopulation jupyter nbconvert world_population_analysis.ipynb
+
+script:
+  - ls -l world_population_analysis.ipynb
```

Comment:
Done: https://github.com/diana-hep/yadage/issues/2

I'll grep the output until then.